### ISSUE TITLE
Is there any way to add branch from svn to git repo after convertation?

### ISSUE THREAD

**Issue by dmarsentev** [nirvdrum/svn2git#187]
Hello! Thanks a lot for this converting tool!

I converted svn repo to git repo
and mirroring svn repo into git repo by "svn2git --rebase".
It processed successfully! 

There are 20 branches in the svn repo and I successfully mirror that svn commits into my git repo.
Suppose, another developer added 21st branch into svn repo.
Can I convert new svn 21st branch smoothly into my git repo to get 21 git branches?

In other words,
can I convert new svn 21-st-branch into my git repo as 21st branch 
to get svn's 21st-branch-commits in my git repo 21st branch  by "svn2git --rebase"?

Thanks!

**Comment by SabareeshSS** [comment]
Yes, you can get the 21st branch by "svn2git --rebase" command on the local git trace folder, which you used to move all the 20 branches.

In precise, on the top of the 20 branches the rebase command performs the incremental migration.
